xml/ftp_banners.xml

```diff
@@ -211,6 +211,8 @@ example.com FTP server (Version:  Mac OS X Server) ready.</example>
     <param pos="0" name="os.vendor" value="Linksys"/>
     <param pos="0" name="os.device" value="WAP"/>
```

Comment:
Is os.device a thing? I thought this was hw.device, but I see os.device everywhere.

Reply:
It _is_ a thing but it is used incorrectly more often than not. Also, in almost every case it should be paired with `hw.device`

Reply:
any suggestions on what to do with this? You marked a few hw.device etc. in #400 that need to be corrected. We probably just need to audit the file specifically for all of those.

Reply:
I'd add `hw.device` to this FP but honestly we need to perform an audit and fix them so either way is fine.